### ISSUE TITLE
Encoding issue

### ISSUE THREAD

**Issue by serdam55** [ravibpatel/AutoUpdater.NET#118]
if I set in the changelog file a string like this: possibilità
I'll get a bad encoding with question mark, could you fix that?

**Comment by ravibpatel** [comment]
@serdam55  Do you mean in in web browser control Where changelog is displayed using provided URL?

**Comment by serdam55** [comment]
yes

**Comment by serdam55** [comment]
nope. I mean when the window with release notes is displayed. There is a bad encoding on it

**Comment by ravibpatel** [comment]
@serdam55 Can you give me your XML file URL?
